src/libs/polycube/src/utils.cpp

```diff
@@ -307,6 +311,49 @@ std::string get_netmask_from_CIDR(const int cidr) {
   return inet_ntoa(addr);
 }
 
+void split_ip_and_prefix(const std::string &ip_and_prefix,
+                        std::string &ip_address, std::string &netmask) {
+  // ip_and_prefix = ip_address/prefix
+  std::istringstream split(ip_and_prefix);
+  std::vector<std::string> info;
+  char split_char = '/';
+  for (std::string each; std::getline(split, each, split_char);
+       info.push_back(each))
+    ;
+  ip_address = info[0];
+  netmask = get_netmask_from_CIDR(std::atoi(info[1].c_str()));
+}
+
+bool address_in_subnet(const std::string &ip, const std::string &netmask,
+                       const std::string &network) {
+  uint32_t ipAddress = ip_string_to_be_uint(ip);
+  uint32_t mask = ip_string_to_be_uint(netmask);
+  uint32_t net = ip_string_to_be_uint(network);
+  if ((ipAddress & mask) == (net & mask))
+    return true;
+  else
+    return false;
+}
+
+std::string get_network_from_ip(const std::string &ip,
+                                const std::string &netmask) {
+  // get the network from ip
+  uint32_t address = ip_string_to_be_uint(ip);
+  uint32_t mask = ip_string_to_be_uint(netmask);
+  uint32_t net = address & mask;
+  char buffer[100];
+  sprintf(buffer, "%d.%d.%d.%d", (net) & 0xFF, (net >> 8) & 0xFF,
+          (net >> 16) & 0xFF, (net >> 24) & 0xFF);
+  std::string network(buffer);
+  return network;
+}
+
+std::string from_int_to_hex(int t) {
```

Comment:
Is the 'from' in the function name needed? It looks to me that the other functions in utils.cpp do not have 'from'.